tox.ini

```diff
@@ -8,6 +8,16 @@ envlist =
     readme
 skip_missing_interpreters = True
 
+[python-cli-options]
+byte-warnings = -b
+byte-errors = -bb
+max-isolation = -E -s -I
```

Comment:
Current docs indicate that `-I` implies `-E` and `-s`. So I think we can cut this down to just `-I`?